### PR TITLE
Pirate ship Remake 

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request

This is no longer a draft, and im sure i'll be making further changes. if you dont like anything, or do, please tell me
---

This PR simply remakes the Pirate ship!
so, whats changed?
well, the pirates themselves are the same, and the starting equipment is mostly the same, except for some changes:
-1 retro laser gun
+1 doublebarrel shotgun
+misc types of loose shotgun shells + spare ammo
-1 bola
and a lot of misc junk and basic equipment has been added
_______

i wont lie, the old shuttle has a LOT of soul, and i'd totally get it if people dont want to see it go. Though do please give it a chance :)

- [x] make the shuttle fly the right way
- [ ] re-balancing if needed

## Why It's Good For The Game
the previous shuttle was very old, and dated. While i wouldnt say that its BAD, i do think it could use a replacement at this point 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

StrongDMM-2024-04-03 19 27 38

image

image

image

image

image

image

(i'll replace the images once the april event is over. all line decals are red/white)
</details>

## Changelog
:cl:
tweak: Remakes Pirate_default.
/:cl:

### CODE REVIEW

File: _maps/shuttles/pirate/pirate_default.dmm [hunk 1]
```diff
@@ -1,1357 +1,2564 @@
...
+	dir = 1
+	},
+/turf/open/floor/wood/big,
+/area/shuttle/pirate)
+"mR" = (
+/mob/living/simple_animal/cow{
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0);
+	desc = "Stolen from a cow genetics lab, she can survive in a vacuum safely, her milk is also delicious!";
+	name = "Ol Betsy"
+	},
```
Comment (EvilDragonfiend) [inline _maps/shuttles/pirate/pirate_default.dmm]: please give pirate faction to cow so turret won't misfire

**Comment by spockye** [comment]
(Not abandoned just yet. Work will resume soon!)

File: _maps/shuttles/pirate/pirate_default.dmm (hunk 1 above)
Comment (spockye) [reply _maps/shuttles/pirate/pirate_default.dmm]: fixed, thank you!

**Comment by spockye** [comment]
Fixed and Ready for review!

**Comment by MarkusLarsson421** [comment]
<details>
<summary>Review</summary>
I'll start of by pointing out that this map is exceptionally high detailed. It may cause issues with future edits or style conflicts compared to other maps. Currently only maintenance is messy and not nearly as messy and detailed as this shuttle. While it looks amazing, it might play out differently and is worth to keep in mind.

Additional, the Captain has now lost its private quarters. While I think it didn't serve much of a practical purpose it still had a thematic purpose.

Having to destroy the wooden barricade every time feels weird. How did they launch the shuttle in the first place? Why did they put up the wooden barricade even?

image

Good that this isn't actual foam plating.

image

Having three stacks of three random items looks weird. Spread them out a tad and vary the amount. Put it in a bin, crate, locker, table, and so on.

image

image

image

image

Corner poster is a bit weird, but that's a minor nitpick.

image

Air fan in walls. Is this intentional?

image

I'd make this a seperate decoration object and put two of them on either side of the wall only visible from one the two sides. And Engineering and the blast door room are missing one each.

image

image

No idea whether the stingbag and frag grenade are cool to give the pirates, someone else should give their opinion on that. I feel like pirates are strong as is.

image

Relatively easy to get to the pirate thrusters and then destroy them with a welder or such.

image

You're just using the "propulsion/left". Just use propulsion, the one the Science shuttle has or left and right to make them connect nicely. No practical difference to my knowledge.

image

What is this supposed to power? Is it just for aesthetics?

image

image

Having the Booze-O-Mat be accessible from both sides of the room is weird. Make the girder a full wall and put it in there instead?

image

Instead of randomly putting up boards, put girders under the boards to make it look like it's a damage which has been repaired in a rush.

image

Maybe a freezer or fridge for the milk to keep 'em cold?

image

Flying is not working as intended.

image

image

Still works without any thrusters...?

image

Shuttle do be massive.

image

Oh... slight error. Could be 

image

image

Air alarm and fire alarm, yet no firedoors.

image

image

Overall there are no scrubbers to vent waste gases.

image

The parrot WILL steal the ammo.

image

image

Incorrect access on the windoors.

image

There are not enough backpacks for the pirates, forcing someone to go without or with a duffel bag is this intended?
Also Pirates can't even open it.

image

image

Can't wrench these to move them, is this intended? Destroying them drops nothing.
image

A bodybag is a great tool for carrying lots of loot!

image

Might wanna blacklist plasma sheets or remove it. Probably remove it. That was just 10 plasma.

image

Would be neat to include some programs here seeing as you can't download any until you reach the station.

image

Perhaps a button outdoor so that they can close it behind themselves? Although I do like this, as it would let crew board the shuttle easier. I'd almost put the button further in so you can only close it whilst inside.

image

Probably want blast doors for the front windows.

image

Currently it is not particularly well protected by the turrets, one of which has to "mine" through the windows in order to hit anyone in the front window.

image

image

Dog beg? It's a parrot. smh

image

</details>

**Comment by spockye** [comment]
turrets not having full cover is intended. pirates shouldnt stay in one place too long, and should be encouraged to hide their ship, or keep an eye on it imo!

on the ship not working, im not sure. it works fine on my side weirdly enough. i'd really appreciate if someone else could try to spawn the pirates in and launch the ship, to see if it JUST works with me <3

**Comment by EvilDragonfiend** [comment]
testmerging by request

**Review by PestoVerde322 — APPROVED** [review]

**Review by EvilDragonfiend — APPROVED** [review]
This seems fine enough. We can make another change in the future if it's necessary.

**Pull request merged**